packages/ns-api/files/ns.backup

```diff
@@ -50,9 +50,21 @@ def create_backup():
             raise RuntimeError('backup failed')
     return file_name
 
+def is_encrypted(file_path):
+    try:
+        # Run the 'file' command on the specified file
+        result = subprocess.run(['file', file_path], capture_output=True, text=True, check=True)
+        # Check if the output contains indications of PGP encryption
+        if "GPG" in result.stdout:
```

Comment:
If for any reason the file name contains `GPG` string, this function will return true even if the file is not really encrypted.